### ISSUE TITLE
security

### ISSUE THREAD

**Issue by anvilvapre** [IceReaper/ORASM#1]
perhaps good to emphasize that the tool should only be used by people who understand the impact. 

you're server side php scripts are - very much - vulnerable to executing arbitrary shell commands.

even with basic authentication in front, some brute forcing and you have server access. 

i.e. you can execute shelll commands, do port scans on localhost, download arbitrary github repo content en execute it. all because the payload is not validated.

**Comment by IceReaper** [comment]
Yeah i know, thats why i added the warning. I just wrote it very quick to automate my server setups with a few clicks to avoid the need to connect via ssh, wgetting stuff, changing start scripts, firing up the servers, managing the screens etc. Otherwise i would have done a bigger setup using an authentication layer, parameter validation, a proper database, mvc setup, template system, multigame support, whatever :D But i was lazy in this case. I am just sharing it as i think someone might find a use for it. Mayble ill enhance this one day, but im currently focusing on openra development, and dont want to spend too much time on a server web interface.
